easybuild/easyconfigs/s/sympy/sympy-1.3_allpython.patch

```diff
@@ -0,0 +1,102 @@
+--- sympy-1.3/sympy/utilities/_compilation/tests/test_compilation.py_orig   2019-01-21 21:47:46.552355000 +0100
```

Comment:
@jose-d Please include a brief description of the patch above this line, and mention author(s)

Reply:
> $ eb --from-pr 7405 --check-style

oh thanks. `--check-style` is the option!